static_files/heimdall_v2/app.toml

```diff
@@ -236,14 +236,20 @@ max-txs = 5000
 [custom]
 # NOTE: This part of the config is similar to heimdall-config.toml (heimdall-v1).
 
-##### RPC and REST config #####
+##### RPC and REST configs #####
 
 # AMQP endpoint.
 amqp_url = "{{.amqp_url}}"
 
 # RPC endpoint for the EL chain.
 bor_rpc_url = "{{.el_rpc_url}}"
 
+# GRPC flag for bor chain.
+bor_grpc_flag = "true"
+
+# GRPC endpoint for bor chain.
+bor_grpc_url = "http://0.0.0.0:{{.grpc_port_number}}"
```

Comment:
GRPC in bor isn't enabled by default. Adding this will lead to the following error:
```
7:05PM ERR unable to connect to bor chain error="rpc error: code = Unimplemented desc = unknown service bor.BorApi"
7:05PM ERR Error occurred while generating milestone proposition error="failed to get latest header" module=server
```

Reply:
Ok, I'll remove that setting then